WcaOnRails/app/controllers/server_status_controller.rb

```diff
@@ -2,6 +2,22 @@
 class ServerStatusController < ApplicationController
   MINUTES_IN_WHICH_A_JOB_SHOULD_HAVE_STARTED_RUNNING = 5
 
+  def self.bad_keys_by_type_by_locale
+    @bad_keys_by_type_by_locale ||= begin
+      bad_keys_by_type_by_locale = {}
```

Comment:
Not in love with `hash` either. I usually call these `result`, but it doesn't quite fit here. And I wanted to avoid yet another thing with the same name.

Yeah, describing the data structure returned in detail is rarely the best naming scheme. What the function is intended for is a better aim.